util/proxy.go

```diff
@@ -62,7 +64,7 @@ func (e *envOnce) init() {
 }
 
 func fromEnvironment() netproxy.Dialer {
```

Comment:
If we have removed `SetMySQLProxyFromEnvironment`, then there is no reason to leave an non-exposed method.

Reply:
make sense.

Reply:
Resolved on 668a3e036f658a7d27a6faab1d40a70526a9e370